<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGroupForm</class>
 <widget class="QWidget" name="AddGroupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #F5F8FA;
color: #14171A;
font-family:&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="addGroupLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="headerTitle">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 18px;
font-weight:500;
</string>
       </property>
       <property name="text">
        <string>Add new group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="formGroup">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>390</width>
         <height>300</height>
        </size>
       </property>
       <property name="title">
        <string>Group Data</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="nameFormWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="styleSheet">
              <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameInput">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	padding: 6px 4px;
	border: 1px solid #aab8c2;
	border-radius: 2px;
}
QLineEdit:hover, QLineEdit:focus{
	border: 1px solid #14171a;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="subjecyFormWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="subjectLabel">
             <property name="styleSheet">
              <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
             </property>
             <property name="text">
              <string>Subject:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="subjectInput">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	padding: 6px 4px;
	border: 1px solid #aab8c2;
	border-radius: 2px;
}
QLineEdit:hover, QLineEdit:focus{
	border: 1px solid #14171a;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="teacherFormWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="teacherLabel">
             <property name="styleSheet">
              <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
             </property>
             <property name="text">
              <string>Teacher:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="teacherInput">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	padding: 6px 4px;
	border: 1px solid #aab8c2;
	border-radius: 2px;
}
QLineEdit:hover, QLineEdit:focus{
	border: 1px solid #14171a;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="footerWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>390</width>
         <height>80</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="successLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: green</string>
          </property>
          <property name="text">
           <string>Group has been successfully added.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="formButtonBox">
          <property name="styleSheet">
           <string notr="true">QPushButton {
background: #616161;
border-radius: 2px;
color: #fff;
padding: 8px 16px;
}</string>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
